src/twisted/web/http_headers.py

```diff
@@ -43,8 +43,8 @@ class Headers:
     """
     Stores HTTP headers in a key and multiple value format.
 
-    Most methods accept L{bytes} and L{unicode}, with an internal L{bytes}
-    representation. When passed L{unicode}, header names (e.g. 'Content-Type')
+    Most methods accept L{bytes} and L{str}, with an internal L{bytes}
+    representation. When passed L{str}, header names (e.g. 'Content-Type')
```

Comment:
It's not really a requirement for this PR, but I feel like we should remove the phrase about the internal representation, because that's… internal and really none of your business as a client of the API.

Reply:
Removed